scripts/package/copy-extensions.sh

```diff
@@ -4,19 +4,10 @@ set -euo pipefail
 cd "$(dirname "${0}")/../.."
 APP_ROOT=$(pwd)
 
-function ensureBuiltinExtensitions() {
-	cd "${APP_ROOT}/lib/vscode"
-	if [ ! -e "extensions/emmet/dist/browser" ]
-	then
-		echo "compile vscode builtin extensions..."
-		yarn gulp compile-web
-	fi
-}
-
 function main() {
-	ensureBuiltinExtensitions
 	cd ${APP_ROOT}
-	mkdir -p dist/static/extensions
+	TARGET="vscode-web-github1s/dist"
+	mkdir -p "${TARGET}/extensions"
```

Comment:
What does this line do?

Reply:
i think it creates a directory for the extensions at the required path.